packages/payment-integration-api/src/payment-integration-selectors.ts

```diff
@@ -72,4 +72,6 @@ export default interface PaymentIntegrationSelectors {
 
     isPaymentDataRequired(useStoreCredit?: boolean): boolean;
     isPaymentMethodInitialized(query: { methodId: string; gatewayId?: string }): boolean;
+
+    getInstruments(): PaymentInstrument[] | undefined;
```

Comment:
It might be better to place this method on the line 37 to keep all instruments related methods in one place